com.sublimetext.three.yaml

```diff
@@ -19,6 +19,10 @@ finish-args:
   - --env=PATH=/app/utils/bin:/app/sublime_merge/bin:/app/bin:/usr/bin
 
 add-extensions:
+  com.sublimetext.three.SublimeMerge:
```

Comment:
Doesn't this need to be `com.sublimemerge.App` to match the app ID of Sublime Merge? Or does this require an extension to bridge the two apps that doesn't exist yet?

Reply:
No, the extension would be a different flatpak, unrelated to `com.sublimemerge.App`.